recipes/tng/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "1.8.2":
+    url: "https://github.com/gromacs/tng/archive/v1.8.2.tar.gz"
```

Comment:
It seems they moved to gitlab sometime ago.... https://gitlab.com/gromacs/tng/-/tags

is there a reason you are referencing the github page?

Reply:
Hmm. I did not know this. I'll change this.